docs/pbc/all/payment-service-provider/202410.0/base-shop/payment-method-strategies.md

```diff
@@ -0,0 +1,82 @@
+            "query_params": {
+               "currency": "currencyName",
+               "locale": "localeName"
+            }
+         }
+      ],
+      "payment_service_provider_data": {
+         "clientId": "AUn5n-...",
+         "merchantId": "3QK...",
+         "isLive": false
+      }
+   }
+}
+```
+
+The main elements of this JSON configuration are the strategy and the scripts.
+
+The `strategy` is set to `express-checkout` to define that the Payment Method should be displayed as a button in the shop.
+
+The `scripts` array contains the URLs to the scripts that are needed to display the button. The URL is used to load the script in the shop. The query_params are used to add key=value pairs to the URL with the actual values. The actual values are taken from the `\SprykerShop\Yves\PaymentAppWidget\Reader\PaymentMethodScriptReaderInterface` which is used in the Storefront to expand the script as needed. The `key` is the name that has to be used as query param key in the URL and the value defines which value should be used. The `currency` and `locale` are the most common values that are used in the scripts.
+
+The `payment_service_provider_data` contains the data that can be used by projects to configure the Payment Method. Those value may differ for each different Payment Method.
+
+## Hosted Payment Page Flow
+
+The default checkout guides a customer through various steps such as the cart, the address, the shipment, and the payment step. After that the customer is shown the Summary Page where he submits the Order. After the order is successfully placed in the Back Office, the Customer is redirected to a Hosted Payment Page where he completes the Payment.
+
+
+## Express Checkout Payment Flow
+
+The Express Checkout Payment Flow belongs to so called PreOrderPayments. PreOrderPayments are payments that are done before the actual order is placed. The Express Checkout Payment Flow is a two-step process. The first step is to authorize the payment and the second step is to capture the payment. The authorization is done when the customer clicks on the button in the shop. The capture is done when the order is placed.
+
+Here the flow is different to the Hosted Payment Page flow. The customer will see e.g. on the Product Detail Page or the Cart Page a button to pay with the Payment Method. When the customer clicks on the button, the payment is initialized and a model is opened where the customer completes the payment. After that the customer is redirected to the Summary Page where he can submit the order. The order is placed in the Back Office and the payment will be captured.   
+
+Additionally, those Express Checkout Payment Methods provide customer data that can be used to prefill the checkout form. This data can contain e.g. customer data, address data or alike.  
```

Comment:
This is general information, but what was written before is technical information.

The page audience is not clear. Most probably the information have to be separated to different pages (for ACP users/project developers and for ACP app developers)

Reply:
Not quite getting it. This is a general page like the API Overview diagrams.